fahclient/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
This whole file should be behind a feature flag, just like the ingress.  In my deployment, I don't need a service, so wouldn't want one created.